src/wizard/rust/index.md

```diff
@@ -9,20 +9,26 @@ To add Sentry to your Rust project you just need to add a new dependency to your
 
 ```toml
 [dependencies]
-sentry = "0.19.0"
+sentry = "{{ packages.version('sentry.rust') }}"
```

Comment:
fairly certain the tags won't work here. Have you tried this?

Reply:
Works fine for me locally

Reply:
Also see https://sentry-docs-git-fix-rust-version-update.sentry.dev/platforms/rust/#install ;-)

Reply:
This is for the wizard when creating a new project in Sentry. It's not the same as the src/platforms content